packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
+
+Here's an example of what a large, centered, red-text paragraph with a blue background and thick padding would look like:
 
 ```html
 <p class="text-lg text-red-600 background-white text-center p-10">
   Hello, world!
 </p>
 ```
 
-It's fairly straightforward once you understand how size, scales and other sequential patterns work. The [Tailwind docs](https://tailwindcss.com/docs) are very helpful in understanding these concepts (and they have an excellent search), and there are plugins for various IDEs ([VS Code](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss), [IntelliJ IDEs](https://plugins.jetbrains.com/plugin/12074-tailwindcss), [Vim](https://github.com/iamcco/coc-tailwindcss)).
+The Tailwind pattern becomes straightforward once you understand how size, scales and other sequential patterns work. The Tailwind docs are very helpful in understanding these concepts (and they have an excellent search), and there are plugins for various IDEs ([VS Code](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss), [IntelliJ IDEs](https://plugins.jetbrains.com/plugin/12074-tailwindcss), [Vim](https://github.com/iamcco/coc-tailwindcss)).
+
+##### Component Classes
+
+When there is a pattern of commonly repeated utility combinations, it may make sense to extract this pattern into its own component or component utility class using Tailwind's `@apply`. A good rule of thumb for this is to consider if a pattern update would be expected to reflect in more than one component.
+
+For example:
+
+```tsx
+// my-component.tsx
+<button className="block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded hover:border-grey-200 hover:bg-grey-100 hover:text-grey-400 hover:transition hover:duration-150 active:border-grey-400 active:bg-grey-100 active:text-grey-400 active:transition active:duration-150">
+  Click here!
+</button>
+
+// my-other-component.tsx
+<a href="#" className="block w-full py-2 text-center mt-6 bg-grey-10 border border-grey-200 transition duration-150 rounded hover:border-grey-200 hover:bg-grey-100 hover:text-grey-400 hover:transition hover:duration-150 active:border-grey-400 active:bg-grey-100 active:text-grey-400 active:transition active:duration-150">
+  Click here!
+</a>
+```
+
+This makes more sense as a new utility class instead of a component because sometimes we style a button and other times a link. Let's extract this instead into `ctas.scss` with a fitting name:
```

Comment:
We should distinguish that this doesn't need to be SCSS (i.e. you can use `@apply` or other PostCSS at-rules without SCSS/nesting). Also, I don't know if it should be noted here or not, but you currently can't import PostCSS files in fxa-react or fxa-admin due to the absence of `postcss-import`.